index.js

```diff
@@ -116,7 +117,7 @@ var Dropzone = React.createClass({
   },
 
   open: function() {
-    var fileInput = React.findDOMNode(this.refs.fileInput);
+    var fileInput = ReactDOM.findDOMNode(this.refs.fileInput);
```

Comment:
Do we actually need ReactDOM here?

We should test to see if `this.refs.fileInput` is enough... That way this project doesn't have a dependency on `ReactDOM`